qtconsole/tests/test_jupyter_widget.py

```diff
@@ -82,3 +82,28 @@ def test_other_output(self):
             '<p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"><br /></p>\n'
             '<p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"><span style=" color:#000080;">In [</span><span style=" font-weight:600; color:#000080;">2</span><span style=" color:#000080;">]:</span> </p></body></html>'
         ))
+
+    def test_copy_paste(self):
```

Comment:
Let's rename this to `test_copy_paste_prompt` so that it's clearer what it does.